src/components/Example2.tsx

```
@@ -0,0 +1,371 @@
+
+function getSuperSecretApiKey(){
+    let importantApiKey = 'FECXS123CXVFDC2332VFDQ.KEY';
+}
+
+const TiMEli_ne() => {
+                                    return 0;
+                            }
+
+
+function spaghetti(){
+    if (true){
```

Biome (1.9.4)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)